docs-v2/spec.yaml

```diff
@@ -1973,3 +2042,42 @@ components:
                 type: string
                 format: date
                 description: Last time it was updated
+    ConnectSession:
+        type: object
+        required:
+            - end_user
+        properties:
+            end_user:
+                type: object
+                required:
+                    - id
+                    - email
+                properties:
+                    id:
+                        type: string
+                        description: Uniquely identifies the end user.
+                    email:
+                        type: string
+                    display_name:
+                        type: string
+            organization:
+                type: object
+                properties:
+                    id:
+                        type: string
+                        description: Uniquely identifies the organization the end user belongs to
+                    display_name:
+                        type: string
+            allowed_integrations:
+                type: array
+                description: Session would only be able to interact with allowed integrations
```

Comment:
I find the wording confusing, maybe something like `Filters which integrations the user can interact with`